custom-connectors/Actsoft/apiDefinition.swagger.json

```diff
@@ -0,0 +1,14408 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "AT&T Workforce Manager",
```

Comment:
@rdameron As per the website name you have provided metadata, The title of you connector can be `"Actsoft Workforce Manager"`. But is `"AT&T Workforce Manager"`, If you are developing a connector for "AT&T" - Can you please share us a documentation which says you have authority to create a connector for AT&T to avoid legal issues in future.

Reply:
Fixed